packages/extension-sdk-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@looker/extension-sdk-react",
-  "version": "22.2.1",
+  "version": "22.2.2",
```

Comment:
a) this should have bumped to 22.4.0 and
b) it should have at least shown up in the manifest.